.github/workflows/update-snapshot.yml

```diff
@@ -7,6 +7,79 @@ on:
   workflow_dispatch:
 
 jobs:
+  update_examples_snapshot:
+    name: Update component examples snapshot by label
```

Comment:
Common logic here as well, wondering what we could do for future refactoring

Reply:
yml is just the worst

Reply:
JSON was bad, so we invented YML.

JSON ain't so bad anymore ;)